R/pkg/R/mllib_utils.R

```diff
@@ -81,9 +80,11 @@ predict_internal <- function(object, newData) {
 #' model <- read.ml(path)
 #' }
 #' @note read.ml since 2.0.0
-read.ml <- function(path, sparkSession = NULL) {
+read.ml <- function(path) {
   path <- suppressWarnings(normalizePath(path))
-  jobj <- callJStatic("org.apache.spark.ml.r.RWrappers", "load", path, sparkSession)
+  sparkSession <- getSparkSession()
+  callJStatic("org.apache.spark.ml.r.RWrappers", "session", sparkSession)
```

Comment:
Here we call ```session``` method to set spark session, the function name is really confusing, may be named as ```setSession``` should be better, but we use the current name from the very beginning.
There is no other places need to be updated, since here is the only place calling ```RWrappers```. Thanks.